src/main/scala/chisel/compatibility.scala

```diff
@@ -0,0 +1,139 @@
+// See LICENSE for license details.
+
+// Allows legacy users to continue using Chisel (capital C) package name while
+// moving to the more standard package naming convention chisel (lowercase c).
+
+package object Chisel {
```

Comment:
Aren't those new chisel3 features? I think it would be nice to keep the new chisel3 features in `chisel`, otherwise we're going to end up maintaining this forever.